packages/babel-plugin-jest-hoist/src/__tests__/__snapshots__/hoistPlugin.test.ts.snap

```diff
@@ -1,5 +1,42 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`babel-plugin-jest-hoist automatic react runtime: automatic react runtime 1`] = `
+
+jest.mock('./App', () => () => <div>Hello world</div>);
+
+      ↓ ↓ ↓ ↓ ↓ ↓
+
+_getJestObj().mock("./App", () => () =>
+  /*#__PURE__*/ _jsxDEV(
+    "div",
+    {
+      children: "Hello world"
+    },
+    void 0,
+    false,
+    {
+      fileName: _jsxFileName,
+      lineNumber: 1,
+      columnNumber: 32
+    },
+    this
+  )
+);
+
+import { jsxDEV as _jsxDEV } from "react/jsx-dev-runtime";
+var _jsxFileName = "";
```

Comment:
one thing we could do is to wrap it in a function and replace usage - similar to what we do with `_getJestObj` now. then we wouldn't have to move it